.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```

Comment:
I don't get this last check. Shouldn't the documentation be added in the same PR? Or if not, open an issue for it? Or the check is when the PR _adds_ documentation?

Reply:
Should we add other options? If it's not a fix nor a feature, like code style changes, refactors, etc.
Maybe instead of listing all possibilities just do:
```
- [ ] Other: fill in the type of change
```